x-pack/plugin/logsdb/src/test/java/org/elasticsearch/xpack/logsdb/SyntheticSourceIndexSettingsProviderLegacyLicenseTests.java

```diff
@@ -98,7 +98,7 @@ public void testGetAdditionalIndexSettingsTsdb() throws IOException {
     }
 
     public void testGetAdditionalIndexSettingsTsdbAfterCutoffDate() throws Exception {
-        long start = LocalDateTime.of(2025, 2, 2, 0, 0).toInstant(ZoneOffset.UTC).toEpochMilli();
+        long start = LocalDateTime.of(2025, 2, 5, 0, 0).toInstant(ZoneOffset.UTC).toEpochMilli();
```

Comment:
2nd is Sunday, 5th is Wed. Is this what we want?

Reply:
This tests checks what settings are returned for tsdb with license after cutoff date.
The cut off date is updated to 2025-02-04 (which is a Tuesday). So I set it to 1 day later. Note this was the case before as well, when cut off date was 2025-02-01.

Reply:
Nm, I hit the wrong button and saw only this change - got confused..